dmp/user-story-all-dmps.ipynb

```diff
@@ -134,10 +134,12 @@
    "outputs": [],
```

Comment:
<p>https://doi.org/10.13039/100000141 this funder_id is from "Division of Ocean Sciences" and has zero management plans, is that the correct DOI ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/datacite/pidgraph-notebooks-python/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Yes, for example 'doi:10.48321/D14S38' has:
`<fundingReference><funderName>Division of Ocean Sciences (nsf.gov)</funderName><funderIdentifier funderIdentifierType="Crossref Funder ID">https://api.crossref.org/funders/100000141</funderIdentifier><awardNumber awardURI="https://www.nsf.gov/awardsearch/showAward?AWD_ID=1356966">1356966</awardNumber><awardTitle>Turbulence-spurred settlement: Deciphering a newly recognized class of larval response</awardTitle></fundingReference>`

The funder is listed in the Organization and repository tables.